task10.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+
+using namespace std;
+
+void printNumberInBinaryForm(int number) {
+    cout << number << " in binary from: ";
+
+    while(number > 0) {
```

Comment:
It doesn't work for the number 4, because it is reversed and it shows 001, instead of 0100